src/mudlet.cpp

```diff
@@ -490,22 +491,26 @@ void mudlet::layoutModules()
             auto itemLocation = new QTableWidgetItem();
             auto itemPriority = new QTableWidgetItem();
             masterModule->setFlags(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled | Qt::ItemIsSelectable);
-            QStringList moduleInfo = pH->mInstalledModules[pModules[i]];
-            masterModule->setText("sync?");
-            if (moduleInfo[1].toInt()) {
-                masterModule->setCheckState(Qt::Checked); //Qt::Checked);
+            QStringList moduleInfo = mpModuleTableHost->mInstalledModules[pModules[i]];
+
+            if (moduleInfo.at(1).toInt()) {
+                masterModule->setCheckState(Qt::Checked);
             } else {
-                masterModule->setCheckState(Qt::Unchecked); //Qt::Checked);
+                masterModule->setCheckState(Qt::Unchecked);
             }
-            masterModule->setToolTip(QString("Checking this box will cause the module to be saved & resync'd across all open sessions."));
+            masterModule->setText(QString());
+            masterModule->setToolTip(QStringLiteral("<html><head/><body><p>%1</p></body></html>").arg(tr("Checking this box will cause the module to be saved and <i>resynchronised</i> across all sessions that share it when the <i>Save Profile</i> button is clicked in the Editor or if it is saved at the end of the session.")));
```

Comment:
It does ___not occur___ if "auto-saving" is set or if "Save as" is used (obviously) or if the lua `saveProfile()` is used even with no arguments...